src/main/kotlin/ecommerce/repository/UserRepository.kt

```diff
@@ -0,0 +1,49 @@
+package ecommerce.repository
+
+import ecommerce.dto.user.UserDTO
+import ecommerce.mapper.UserRowMapper
+import org.springframework.jdbc.core.JdbcTemplate
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert
+import org.springframework.stereotype.Repository
+
+@Repository
+class UserRepository(
+    private val jdbcTemplate: JdbcTemplate,
+    private val userRowMapper: UserRowMapper,
+) {
+    fun create(user: UserDTO): Long {
+        val insert =
+            SimpleJdbcInsert(jdbcTemplate)
+                .withTableName("users")
+                .usingGeneratedKeyColumns("id")
+
+        val parameters =
+            mapOf(
+                "email" to user.email,
+                "password" to user.password,
```

Comment:
Yes, as you mentioned, please try implementing password encoding when you have time 🙂